### DISCUSSION TITLE
Row based access control [Jul 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6061] [Discord Threads]
> 🙋‍♂️ **olmoj**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### RBAC- not supported currently.    
####    
#### You can create views & share link, but this will only give read only access to those with link.  
#  
> 🙋‍♂️ **olmoj**  
#### Would it be possible to add it as new feature in roadmap?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1116749946062897224)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
